tests/dragonfly/connection_test.py

```diff
@@ -0,0 +1,176 @@
+import asyncio
+import aioredis
+import async_timeout
+#from conftest import DATABASE_INDEX
+
+
+def verify_response(monitor_response: dict, key: str, value: str) -> bool:
+    if monitor_response is None:
+        return False
+    if monitor_response["db"] == 1 and monitor_response["client_type"] == "tcp":
+        return key in monitor_response["command"] and value in monitor_response["command"]
+    else:
+        return False
+
+
+async def monitor_cmd(mon: aioredis.client.Monitor, messages: dict):
+    success = None
+    async with mon as monitor:
+        try:
+            for key, value in messages.items():
+                try:
+                    async with async_timeout.timeout(1):
+                        response = await monitor.next_command()
+                        success = verify_response(
+                            response, key, value)
+                        if not success:
+                            return False, f"failed on the verification of the message {response} at {key}: {value}"
+                        await asyncio.sleep(0.01)
+                except asyncio.TimeoutError:
+                    pass
```

Comment:
why do you return True in case of TimeoutError?

Reply:
I don't retunr, it would only return after it run all the expected messages, the pass in this case is only to ignore this exception, it means that we still don't have a message after this timeout.